CHANGELOG.md

```diff
@@ -6,9 +6,13 @@ The public API of this library consists of the functions declared in file
 [h3api.h.in](./src/h3lib/include/h3api.h.in).
 
 ## [Unreleased]
+
+## [3.5.1] - 2019-08-12
```

Comment:
With new features added, shouldn't this be 3.6.0 instead?

Reply:
Oops, yes I believe it should.